### ISSUE TITLE
Input sanitization sniff flags unseting an input var

### ISSUE THREAD

**Issue by JDGrimes** [WordPress-Coding-Standards/WordPress-Coding-Standards#187]
If you unset elements from an input variable, it will be flagged as being not validated and not sanitized:

``` php
unset( $_POST['some_var'] );
```

Although this is probably not the best idea to start with, it isn't unsafe of itself.
